packages/core/src/RPA/core/windows/__init__.py

```diff
@@ -17,6 +17,7 @@ class WindowsElementsMixin:
 
     def __init__(self, locators_path: Optional[str] = None):
         self.logger = logging.getLogger(__name__)
+        self.list_processes = True
```

Comment:
Seems like a good way to control the library when we're about to decide if we want to list processes by PIDs or not. 

But who/what sets this `list_processes` attribute (as `False` for e.g.) over the library object instance? And do you have a test/example showcasing the usage?

Reply:
Was planning to adding keyword to Windows library to toggle this setting but obviously it requires first that this value exists in the `core`.